### ISSUE TITLE
Disambiguation issue (selector).

### ISSUE THREAD

**Issue by jfjlaros** [mutalyzer/mutalyzer#82]
Descriptions [`NM_002001.2(NP_001992.1):p.Glu100_Ser101insAla`](https://mutalyzer.nl/normalizer/NM_002001.2(NP_001992.1):p.Glu100_Ser101insAla) and [`NM_002001.2:p.Glu100_Ser101insAla`](https://mutalyzer.nl/normalizer/NM_002001.2:p.Glu100_Ser101insAla) are equivalent, but do not result in the same disambiguated description.

**Comment by mihailefter** [comment]
@jfjlaros: Do you perhaps know if there is any preference from the HGVS perspective for one of them?

**Comment by jfjlaros** [comment]
No, but the fist one seems to be consistent with the usage of selectors when using a genomic reference.

**Comment by mihailefter** [comment]
@jfjlaros the same output is provided now for both input descriptions. I think this can be closed now.
